sources/packages/api/src/route-controllers/program-info-request/models/program-info-request.dto.ts

```diff
@@ -1,15 +1,48 @@
-import { IsNotEmpty, IsInt, Min, IsOptional } from "class-validator";
-import { SaveOfferingDTO } from "../../education-program-offering/models/education-program-offering.dto";
-import { ProgramInfoStatus } from "../../../database/entities";
+import {
+  IsNotEmpty,
+  IsInt,
+  Min,
+  IsOptional,
+  IsPositive,
+} from "class-validator";
+import {
+  OfferingStatus,
+  OfferingTypes,
+  ProgramInfoStatus,
+} from "../../../database/entities";
 import { OfferingIntensity } from "../../../database/entities/offering-intensity.type";
 
-export interface CompleteProgramInfoRequestDto extends SaveOfferingDTO {
-  selectedProgram?: number;
-  selectedOffering?: number;
+export class CompleteProgramInfoRequestAPIInDTO {
+  @IsPositive()
+  selectedOffering: number;
 }
 
-export interface GetProgramInfoRequestDto
-  extends CompleteProgramInfoRequestDto {
+export interface ProgramInfoRequestAPOutDTO {
```

Comment:
Do we need all these properties returned after we removed the offering from the UI?
The PIR UI looks like the below one now, right? Can we ensure that we are not returning more data that is needed?
![image](https://user-images.githubusercontent.com/61259237/166336553-81af99a7-d47a-4737-83fd-12c5b4cfb55e.png)

Reply:
Are the below-highlighted ones still used?
![image](https://user-images.githubusercontent.com/61259237/166552836-b097b32a-e8cd-4602-848a-c150fe4a9d42.png)

Reply:
Yes, When I remove either of the selected attributes, it breaks the rendering of the formio form.

Reply:
As we talked, let's leave these ones.